packages/mermaid/src/rendering-util/rendering-elements/shapes/util.ts

```diff
@@ -30,19 +39,19 @@ export const labelHelper = async (parent, node, _classes) => {
     label = typeof node.label === 'string' ? node.label : node.label[0];
   }
 
-  let text;
-  text = await createText(labelEl, sanitizeText(decodeEntities(label), getConfig()), {
+  const text = await createText(labelEl, sanitizeText(decodeEntities(label), getConfig()), {
     useHtmlLabels,
-    width: node.width || getConfig().flowchart.wrappingWidth,
+    width: node.width || getConfig().flowchart?.wrappingWidth,
+    // @ts-expect-error -- This is currently not used. Should this be `classes` instead?
     cssClasses: 'markdown-node-label',
```

Comment:
This value is ignored, maybe it should be the following instead?

```ts
    // @ts-expect-error -- This is currently not used. Should this be `classes` instead?
    classes: 'markdown-node-label',
```

I didn't want to change it in this PR, since this one would probably cause visual regression changes, so having a separate PR where we decide to fix it/delete it would probably be best.